cli/lib/kontena/cli/registry/remove_command.rb

```diff
@@ -8,11 +8,14 @@ def execute
       require_api_url
       token = require_token
       confirm unless forced?
+      name = 'registry'
 
-      registry = client(token).get("services/#{current_grid}/registry") rescue nil
-      abort("Docker Registry service does not exist") if registry.nil?
+      registry = client(token).get("services/#{current_grid}/#{name}") rescue nil
+      abort("#{name.colorize(:cyan)} service does not exist") if registry.nil?
```

Comment:
(see previous comments) To unify things CLI should use the abort if something does not exist, e.g. the scope of the command can not be performed. abort() should output red "ERROR:" prefix